tests/registry/test_core_contracts.py

```diff
@@ -0,0 +1,120 @@
+import pytest
+from tracecat_registry.base.core.contracts import require
+
+
+@pytest.mark.parametrize(
+    "lhs, rhs, condition",
+    [
+        # Threat score comparisons
+        (85, 75, ">"),
+        (60, 60, ">="),
+        (30, 50, "<"),
+        # IP address checks
+        ("192.168.1.1", ["192.168.1.1", "10.0.0.1"], "in"),
+        ("8.8.8.8", ["192.168.1.1", "10.0.0.1"], "not in"),
+        # Status code validation
+        (200, 200, "=="),
+        (403, 200, "!="),
+        # Alert severity matching
+        ("critical", "critical", "=="),
+        ("high", "medium", "!="),
+        # Malware hash verification
+        ("e1234...abc", "e1234...abc", "is"),
+        ("abc123", ["abc123", "def456"], "in"),
+        # Domain blocklist check
+        ("malicious.com", ["malicious.com", "evil.com"], "in"),
+        ("good.com", ["malicious.com", "evil.com"], "not in"),
+        # Process state validation
+        (None, None, "is"),
+        ("running", ["running", "suspended"], "in"),
+        # Incident count thresholds
+        (5, 3, ">"),
+        (10, 10, ">="),
+        (2, 5, "<"),
+    ],
+)
+def test_require_pass(lhs, rhs, condition):
+    assert require(lhs, rhs, condition) is True
```

Comment:
if we don't return anything, no need to assert True here. Test will pass if no errors thrown. same for other tests

Reply:
Ignore